scripts/build/docker-cuda-ndebug.sh

```diff
@@ -22,7 +22,7 @@ cmake -G Ninja \
   -DBUILD_SHARED_LIBS:BOOL=OFF \
   -DCMAKE_BUILD_TYPE:STRING="Release" \
   -DCMAKE_CUDA_ARCHITECTURES:STRING="70" \
-  -DCMAKE_CUDA_FLAGS:STRING="--use_fast_math" \
+  -DCMAKE_CUDA_FLAGS:STRING="--use_fast_math -Xnvlink --suppress-stack-size-warning -Werror" \
```

Comment:
Rather than the various build scripts, wouldn't this one be better off inside the CMakeLists.txt (i.e. somehow  the default for all builds)?

Reply:
Ideally `VecGeom` would add `"$<$<COMPILE_LANGUAGE:CUDA>:"SHELL: -Xnvlink --suppress-stack-size-warning">` to the public compile options as long as it's needed. I think you're right that for now we should just add it as part of the main celeritas cmake.

Reply:
Good suggestion, done! Maybe @mrguilima or @drbenmorgan could upstream the change:
```diff
+    # Suppress warnings from virtual function calls in RDC
+    target_compile_options(VecGeom::vecgeomcuda
+      INTERFACE "$<$<COMPILE_LANGUAGE:CUDA>:SHELL: -Xnvlink --suppress-stack-size-warning>"
+    )
```